src/Evolution/Systems/GrMhd/GhValenciaDivClean/FiniteDifference/ReconstructWork.tpp

```diff
@@ -344,8 +361,12 @@ void reconstruct_fd_neighbor_work(
         }
       });
 
-  spacetime_vars_for_grmhd(vars_on_face);
-  ValenciaDivClean::fd::compute_conservatives_for_reconstruction(vars_on_face,
-                                                                 eos);
+  if constexpr (tmpl::size<SpacetimeTagsToReconstruct>::value != 0) {
```

Comment:
Q. Are we planning to use flexible set of metric related tags for reconstruction? Can you give an example of situations where we reconstruct different set of metric vars?

Reply:
One would be reconstructing lapse, shift, and spatial metric instead of the spacetime metric. Lapse, in particular, since that must be positive